app/app.go

```diff
@@ -456,7 +456,7 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 			params.NewAppModule(app.ParamsKeeper),
 			evidence.NewAppModule(app.EvidenceKeeper),
 			feegrantmodule.NewAppModule(appCodec, app.AccountKeeper, app.BankKeeper, app.FeeGrantKeeper, app.interfaceRegistry),
-			authzmodule.NewAppModule(appCodec, app.AuthzKeeper, app.AccountKeeper, app.BankKeeper, app.interfaceRegistry),
+			// authzmodule.NewAppModule(appCodec, app.AuthzKeeper, app.AccountKeeper, app.BankKeeper, app.interfaceRegistry), // enable after updating sdk version v0.46
```

Comment:
why? when is this supposed to happen?

Reply:
We need authz - it's already used and it could break some existing flows.

Reply:
I'm confused why simulation tests were passing before and now the following error occurs when uncommented:

```
    simulate.go:300: error on block  26/100, operation (8/738) from x/authz:
        failed to execute message; message index: 0: requested amount is more than spend limit: insufficient funds [regen-network/cosmos-sdk@v0.44.2-regen-1/types/errors/errors.go:273]
        Comment: failed to execute message; message index: 0: requested amount is more than spend limit: insufficient funds
```

Reply:
@clevinson and I dug into this a bit and we were trying to figure out if this was in fact an issue with the authz module. The error is `insufficient funds` so we were thinking it might be related to the basket fee not being properly accounted for. The simulation tests are successful with authz uncommented and `SimulateMsgCreate` commented out.

Reply:
@clevinson and I dug into this a bit and we were trying to figure out if this was in fact an issue with the authz module. The error is `insufficient funds` so we were thinking it might be related to the basket fee. We were able to successfully run the simulation tests with authz uncommented and skipping `SimulateMsgCreate`.

Reply:
Yes. The issue is with `authz` simulation. We should ignore `MsgExec` simulation if a grantee tries to spend more than the allowed spend limit. I've raised a PR on the SDK to fix this.

Reply:
Great! Thanks for confirming. https://github.com/cosmos/cosmos-sdk/pull/11252